src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/GaugeWave.hpp

```diff
@@ -0,0 +1,164 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
```

Comment:
I'm confused by why partial derivative is needed

Reply:
This must be left over from copied code

Reply:
Never mind, `PartialDerivatives.hpp` declares the deriv tag, which I am using in e.g. 
~~~~
  using DerivLapse = ::Tags::deriv<gr::Tags::Lapse<DataType>,
                                   tmpl::size_t<volume_dim>, Frame::Inertial>;
~~~~